### PR TITLE
Site: Update community links CI and README.md CI

### PR DESCRIPTION
**Pull request by snazy**
Unifying the badges styles

### CODE REVIEW

**Comment by snazy** [comment]
Current:

image

image

**Comment by snazy** [comment]
Change in this PR:

image

image

image

**Review by dimas-b — APPROVED** [review]

File: site/docs/index.md [hunk 1]
```diff
@@ -9,9 +9,12 @@ template: "home.html"
...
-  
 
 Get in touch via our [Google Group](https://groups.google.com/g/projectnessie) and
 our [Slack Channel](mailto:slack-subscribe@projectnessie.org) and follow us on
 [Twitter](https://twitter.com/projectnessie). Nessie source code, code contributions
 and bug reports are on [GitHub](https://github.com/projectnessie/nessie/).
+
+[![Google Group](https://img.shields.io/badge/Google_Group-white.svg?logo=google&style=for-the-badge&color=eeeeee&logoColor=gray)](https://groups.google.com/g/projectnessie)
+[![Twitter](https://img.shields.io/badge/-Twitter-white?logo=twitter&style=for-the-badge&color=eeeeee&logoColor=gray)](https://twitter.com/projectnessie)
+[![GitHub](https://img.shields.io/badge/-GitHub-black?logo=github&style=for-the-badge&color=eeeeee&logoColor=gray)](https://github.com/projectnessie/nessie/s)
```
Comment (dimas-b) [inline site/docs/index.md:20]: missed this in review: the extra `s` in the URL causes 404 :(

Comment (dimas-b) [reply site/docs/index.md:20]: #6167

**Comment by XN137** [comment]
was `site/__pycache__` commited intentionally?
if not we should remove it and add `__pycache__` to gitignore WDYT?

**Comment by snazy** [comment]
Oops - nope. #6201

**Pull request merged**